app/assets/stylesheets/application.css.scss

```diff
@@ -129,6 +145,20 @@ td, th {
   }
 }
 
+h1 {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.